user_sync/rules.py

```diff
@@ -154,15 +162,14 @@ def run(self, directory_groups, directory_connector, umapi_connectors):
             load_directory_stats.log_start(logger)
             self.read_desired_user_groups(directory_groups, directory_connector)
             load_directory_stats.log_end(logger)
-            should_sync_umapi_users = True
-        else:
-            # no directory users to sync with
-            should_sync_umapi_users = False
 
-        umapi_stats = JobStats("Sync Umapi", divider="-")
+        umapi_stats = JobStats('Sync Umapi' if self.sync_umapi else 'Push to Umapi', divider="-")
```

Comment:
Ha!  Great minds think alike.  I originally changes this to UMAPI but then I thought "Oh, no, Phil will kill me if I make a cosmetic change in the reports, because he will have to redo the docs and will worry that there are customers searching on that string!" :)  I am HAPPY to change this occurrence and will do so right now!